### PR TITLE
chore: add missing translation

### LINKED ISSUE #7421: Add missing translation
https://github.com/calcom/cal.com/blob/0ec71e52eff2ac8bdfd4bdd4616c6704f9f5aaeb/packages/features/ee/workflows/components/EventWorkflowsTab.tsx#L69

```javascript

        const message = `${err.statusCode}: ${err.message}`;
        showToast(message, "error");
      }
      if (err.data?.code === "UNAUTHORIZED") {
        // TODO: Add missing translation
        const message = `${err.data.code}: You are not authorized to enable or disable this workflow`;
        showToast(message, "error");
      }
    },
  });


```

### PR DESCRIPTION
**Pull request by iamsriranga**
## What does this PR do?

Fixes #7421

**Environment**: Staging(main branch) / Production

## Type of change

- Chore (refactoring code, technical debt, workflow improvements)

## How should this be tested?

- [ ] Test A
- [ ] Test B

## Checklist

- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't performed a self-review of my own code and corrected any misspellings
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

File: packages/features/ee/workflows/components/EventWorkflowsTab.tsx [hunk 1]
```diff
@@ -68,9 +68,8 @@ const WorkflowListItem = (props: ItemProps) => {
         showToast(message, "error");
       }
       if (err.data?.code === "UNAUTHORIZED") {
-        // TODO: Add missing translation
         const message = `${err.data.code}: You are not authorized to enable or disable this workflow`;
-        showToast(message, "error");
+        showToast(t(message), "error");
```
Comment (Udit-takkar) [inline packages/features/ee/workflows/components/EventWorkflowsTab.tsx:72]: this is wrong approach. i don't think there is a key in translations

**Comment by PeerRich** [comment]
that's not how we can translate error messages unfortunately. Thank you for trying and contributing!

**Comment by saideepesh000** [comment]
Hey @PeerRich @Udit-takkar, would like to know more about this, since it's a dynamic string, how can we add it to translation strings?

**Comment by Udit-takkar** [comment]
@saideepesh000 error.code.message was UNAUTHORISED ( see Line 71) so you can just create a new key in  en.json with that complete message string as value.

**Pull request closed without merging**